subsys/mpsl/cx/CMakeLists.txt

```diff
@@ -4,24 +4,24 @@
 # SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
 #
 
-if (CONFIG_MPSL_CX_3WIRE OR
-    CONFIG_MPSL_CX_1WIRE OR
-    CONFIG_MPSL_CX_BT_1WIRE OR
-    CONFIG_MPSL_CX_NRF700X OR
-    CONFIG_MPSL_CX_SOFTWARE)
-    zephyr_library()
+if(CONFIG_MPSL_CX_3WIRE OR
+  CONFIG_MPSL_CX_1WIRE OR
+  CONFIG_MPSL_CX_BT_1WIRE OR
+  CONFIG_MPSL_CX_NRF700X OR
+  CONFIG_MPSL_CX_SOFTWARE)
```

Comment:
these can remain indented by a further space as the indent is for alignment with the previous line